compiler/rustc_passes/src/check_attr.rs

```diff
@@ -1490,6 +1490,7 @@ fn check_invalid_crate_level_attr(tcx: TyCtxt<'_>, attrs: &[Attribute]) {
         sym::automatically_derived,
         sym::start,
         sym::main,
```

Comment:
`sym::main` can be removed from this list.

Reply:
Done.